apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx

```diff
@@ -60,7 +63,10 @@ export const getIndictmentIntroductionAutofill = (
         prosecutorsOffice?.name?.toUpperCase(),
         `\n\n${formatMessage(strings.indictmentIntroductionAutofillAnnounces)}`,
         `\n\n${formatMessage(strings.indictmentIntroductionAutofillCourt, {
-          court: court?.name?.replace('dómur', 'dómi'),
+          court:
+            court && court.name
+              ? applyDativeCaseToCourtName(court.name)
```

Comment:
If `court` or `court.name` is not set (should never happen btw), then the string will be `Að höfða ber sakamál fyrir héraðsdómi á hendur` and not `Að höfða ber sakamál fyrir Héraðsdómi Reykjavíkur á hendur`